pkg/datapath/loader/base.go

```diff
@@ -317,11 +317,6 @@ func (l *Loader) Reinitialize(ctx context.Context, o datapath.BaseProgramOwner,
 	case option.Config.TunnelingEnabled():
 		mode = tunnelMode
 		encapProto = option.Config.TunnelProtocol
-	case option.Config.EnableHealthDatapath:
-		mode = option.DSRDispatchIPIP
```

Comment:
This statement has gone missing in this PR. I'll push my local branch with this readded in a bit.

Reply:
I figured it's no longer needed given it sets a variable that's used as the input arg for init.sh with a value that's no longer used in init.sh after this PR.